src/main/java/edu/kis/vh/nursery/listAndArray/IntLinkedList.java

```diff
@@ -1,11 +1,13 @@
-package edu.kis.vh.nursery.list;
+package edu.kis.vh.nursery.listAndArray;
 
-public class IntLinkedList {
+public class IntLinkedList implements IntStackList {
 
+	private final int ifEmpty = 0;
```

Comment:
Wrong convention. Constant should be written with an uppercase, e.g., IF_EMPTY).
This constant can be shared with IntArrayStack and should be pushed to the interface!